### PR TITLE
Fetch basic event

### PR DESCRIPTION
**Pull request by jebbinSCL**
Currently doing most of the work in Program.fs for my ease of use. I will split in a later pull request.

### CODE REVIEW

File: Publish/Program.fs [hunk 1]
```diff
@@ -1,25 +1,62 @@
...
-    let contentPairs = 
-        [|
-            new KeyValuePair<string, string>("name","Test event from Publish 1")
-            new KeyValuePair<string, string>("description","Test event from Publish 1")
-        |]
-    let content = new FormUrlEncodedContent(contentPairs)
-    let response = client.PostAsync(uri,content).Result
-    let responseContent = response.Content.ReadAsStringAsync().Result
-    responseContent
+module JsonHttpClient =
```
Comment (jebbinSCL) [inline Publish/Program.fs]: Currently working in one file for my ease of use for now, I will split in a later pull request.

File: Publish/Config.fs [hunk 2]
```diff
@@ -10,4 +10,8 @@ let getConfigValue (key : string) =
     else 
         value
 
+let getUriConfigValue (key : string) = getConfigValue key |> Uri
```
Comment (pallenSL) [inline Publish/Config.fs]: A nicer API would be Config.getString and Config.getUri

File: Publish/Program.fs (hunk 1 above)
Comment (pallenSL) [inline Publish/Program.fs]: Should I do my reviewing on a later pull request?  ;)

Comment (jebbinSCL) [inline Publish/Program.fs]: It will just be that the modules will be in separate files, rather than in line.

Comment (jebbinSCL) [inline Publish/Program.fs]: Also, I have other branches based on this, so it will be really hard to merge if I split this now.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**